KoPrivateGPT/llm/basic.py

```diff
@@ -3,22 +3,20 @@
 
 import openai
 
-from KoPrivateGPT.DB.base import BaseDB
 from KoPrivateGPT.llm.base import BaseLLM
 from KoPrivateGPT.retrieval.base import BaseRetrieval
 from KoPrivateGPT.schema import Passage
 
 
 class BasicLLM(BaseLLM):
-    def __init__(self, retrieval: BaseRetrieval, db: BaseDB, model_name: str = "gpt-3.5-turbo", api_base: str = None,
+    def __init__(self, retrieval: BaseRetrieval, model_name: str = "gpt-3.5-turbo", api_base: str = None,
```

Comment:
You must edit RerankLLM and ViscondeLLM like this, too. (Didn't you merged latest main, yet?)